src/tools/android/java/com/google/devtools/build/android/ManifestMergerAction.java

```diff
@@ -176,8 +187,7 @@ private static Path removePermissions(Path manifest, Path outputDir)
       }
     }
     // Write resulting manifest to the output directory, maintaining full path to prevent collisions
-    Path output = outputDir.resolve(manifest.toString().replaceFirst("^/", ""));
```

Comment:
This was failing on windows machines causing the original files to be modified in place.

Reply:
It will be easier to import this PR if we can evaluate this change separately. Is it right that this was an existing problem and not needed for merging permissions?

Reply:
Correct. I can pull this out into another PR that can be imported independently of this, but this PR will need to be rebased against that for the tests to pass.

Reply:
Pulled this out into https://github.com/bazelbuild/bazel/pull/13760

Reply:
@ted-xie The tests for this wont pass on windows until the other PR lands. I'll rebase this once that lands.